R/RLearner_classif_fdaFDboost.R

```diff
@@ -0,0 +1,116 @@
+#' @export
+makeRLearner.classif.fdaFDboost = function() {
```

Comment:
Did we decide to call it fdaFDboost instead of FDboost.

Reply:
I think having the "fda" prefix for the FDA learners make it a consistent naming convention. Since we have "fdaknn",  if all learners of FDA have the same prefix, we could easily find out all the fda learners, which is better for maintainence

Reply:
```listLearners(properties = "functionals")```
Does the job aswell